org/postgresql/core/Oid.java

```diff
@@ -83,7 +83,8 @@
     public static String toString(int oid) {
         try {
             Field[] fields = Oid.class.getFields();
-            for (Field field : fields) {
```

Comment:
+1 for arrays (I was mistaken)

Although, in my experience HotSpot is rarely able to optimize iterators on lists.